content/en/docs/setup/platform-setup/oci/index.md

```diff
@@ -11,35 +11,88 @@ owner: istio/wg-environments-maintainers
 test: no
 ---
 
-This page was last updated January 4, 2019.
+This page was last updated September 20, 2021.
 
 {{< boilerplate untested-document >}}
```

Comment:
Was this document manually tested to verify it works? I suspect it was. Then we may want to remove this template as well.

Reply:
It was tested by Oracle, yes.